### PR TITLE
adds the skeleton for render file

### PR DESCRIPTION
**Pull request by Karyum**
THE SKELLY MAN

### CODE REVIEW

File: render.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var render = (function() {
+
+  var MovieInfo = function(response) {
```
Comment (RQ16) [inline render.js]: The function name "MovieInfo" should start with  a lower case character. Try changing it to "movieInfo"

File: render.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  }
+
+  var MovieGIF = function(response) {
```
Comment (RQ16) [inline render.js]: The function name "MovieGIF" should start with a lower case character. Try changing it to "movieGIF"

File: render.js (hunk 1 above)
Comment (mattlub) [reply render.js]: I would suggest maybe calling it renderMovieInfo... but this is open to discussion.

Comment (Karyum) [reply render.js]: thanks for the heads up its fixed now

File: render.js (hunk 2 above)
Comment (Karyum) [reply render.js]: Done

File: render.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+var render = (function() {
```
Comment (mattlub) [inline render.js]: should be capitalised please: Render (for consistency)

File: render.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  }
+
+  var renderMovieGIF = function(response) {
+
+  }
+
+  return {
+    MovieInfo: MovieInfo,
+    MovieGIF: MovieGIF
```
Comment (mattlub) [inline render.js]: you need to also update the return object since you have changed the function names.

**Review by mattlub — CHANGES_REQUESTED** [review]

**Review by mattlub — APPROVED** [review]

**Pull request merged**